### ISSUE TITLE
Unable to add new item to vault

### ISSUE THREAD

**Issue by cmvanb** [bitwarden/clients#8722]
### Steps To Reproduce

1. Open Bitwarden desktop client
2. Login with master password
3. Click `+` button to add a new item
4. Enter any text into the `Name` field
5. Click the save button
6. Observe an error toast with the error: `An error has occurred. Failed to decode access token: JWT must have 3 parts`

### Expected Result

I expected to be able to add a new item to my vault.

### Actual Result

I was prevented from adding a new item to my vault.

### Additional Context

Arch Linux installed package details:
```
pacman -Qi bitwarden
Name            : bitwarden
Version         : 2024.4.0-1
Description     : A secure and free password manager for all of your devices
Architecture    : x86_64
URL             : https://github.com/bitwarden/clients/tree/master/apps/desktop
Licenses        : GPL3
Groups          : None
Provides        : None
Depends On      : electron28  libnotify  libsecret  libxtst  libxss  libnss_nis
Optional Deps   : None
Required By     : None
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 34.30 MiB
Packager        : Alexander Epaneshnikov <alex19ep@archlinux.org>
Build Date      : Thu 11 Apr 2024 05:45:13 PM CEST
Install Date    : Fri 12 Apr 2024 01:02:31 PM CEST
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature
```

### Operating System

Linux

### Installation method

Other

### Build Version

2024.4.0-1

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Krychaz** [comment]
Hello there, 

We have just updated our Snap releases to fix this issue. Please update to the latest version and try again.

**Comment by cmvanb** [comment]
Hello @Krychaz 

I don't use snaps. I use the arch linux packages.

Could you confirm for me which version of bitwarden has fixed this issue? The latest version available to me is `2024.4.0-1`, in which it is broken.

**Comment by jmswick** [comment]
I am still getting the JWT error in the `latest/stable:    2024.4.0 2024-04-11` snap that just became available.

**Comment by antonlydell** [comment]
I am having the same issue with the snap package on Ubuntu 23.10.

Bitwarden Snap information:
Version 2024.4.0
Shell 28.2.8
Renderer 120.0.6099.291
Node 18.18.2
Architecture x64

**Comment by keilmillerjr** [comment]
> Hello there,
> 
> We have just updated our Snap releases to fix this issue. Please update to the latest version and try again.

Which commit addresses this issue? There are no commits mentioning "JWT" or "token" for this month.

**Comment by guktib** [comment]
I have the same error here, just like @cmvanb: The error occurs not only when saving new entries but also when I log out and try to log back in. When querying the verification code, the same error message appears, and I can no longer open my vault.

I'm also on ArchLinux and has the same version as @cmvanb.

![bitwarden_decode-error_20240413](https://github.com/bitwarden/clients/assets/66076942/6cc1ed43-a59b-44fe-a4b4-3447460b328b)

**Comment by trmartin4** [comment]
The issue can be resolved by granting the Bitwarden application access to the`password-manager-service`.  The updated Linux desktop version includes an updated description on the snap store [here](https://snapcraft.io/bitwarden):

> Bitwarden requires access to the `password-manager-service` for secure storage. Please enable it through permissions or by running `sudo snap connect bitwarden:password-manager-service`.

We have also updated the Linux installation instructions [here](https://btwrdn.com/install-snap).

Unfortunately this access cannot be automatically granted, so the command needs to be run in order to grant the application the appropriate permissions.

**Comment by keilmillerjr** [comment]
> The issue can be resolved by granting the Bitwarden application access to the`password-manager-service`. The updated Linux desktop version includes an updated description on the snap store [here](https://snapcraft.io/bitwarden):
> 
> > Bitwarden requires access to the `password-manager-service` for secure storage. Please enable it through permissions or by running `sudo snap connect bitwarden:password-manager-service`.
> 
> We have also updated the Linux installation instructions [here](https://btwrdn.com/install-snap).
> 
> Unfortunately this access cannot be automatically granted, so the command needs to be run in order to grant the application the appropriate permissions. Please be sure that the desktop client is logged out when granting this permission, or log out and back in after doing so.

Snap is 100% irrelevant to this post. It worked prior on the arch repo. What changed? We are not using snaps. Snaps suck.

**Comment by cmvanb** [comment]
> Regarding the non-snap issues, can anyone check if [this solution](https://github.com/bitwarden/clients/issues/8651#issuecomment-2051200368) fixes the issues? We're using `libsecret` for secure storage on linux, but I have a feeling many of our users don't have a Secret Service implementation because (understandably) they use Bitwarden.

By rolling back to `2024.3.0` and removing the contents of `~/.config/Bitwarden`, I got the desktop app working again. Thank you.

I should note that not every Arch user will have easy access to this older package.

You are correct that I am not running a libsecret implementation on my system. Is that a new dependency? It appears a bit awkward for BW to depend on libsecret, as it is itself a password manager.

**Comment by guktib** [comment]
I agree with the comment from @cmvanb: _"Is that a new dependency? It appears a bit awkward for BW to depend on libsecret, as it is itself a password manager."_

Why do we need an additional (new?) dependency on a library that (seemingly) fulfills the same task that Bitwarden should have?!

**Comment by uhlwoogi** [comment]
Also seeing this issue with both the Snap package and the appimage on Kubuntu 22

**Comment by sethforprivacy** [comment]
> The issue can be resolved by granting the Bitwarden application access to the`password-manager-service`. The updated Linux desktop version includes an updated description on the snap store [here](https://snapcraft.io/bitwarden):
> 
> > Bitwarden requires access to the `password-manager-service` for secure storage. Please enable it through permissions or by running `sudo snap connect bitwarden:password-manager-service`.
> 
> We have also updated the Linux installation instructions [here](https://btwrdn.com/install-snap).
> 
> Unfortunately this access cannot be automatically granted, so the command needs to be run in order to grant the application the appropriate permissions. Please be sure that the desktop client is logged out when granting this permission, or log out and back in after doing so.

Doing this fixed the Snap for me on Ubuntu 22.04 FWIW.

**Comment by fernandominardi** [comment]
Having the same issue here. In my case running the `sudo snap connect bitwarden:password-manager-service` command does not solve the problem.
```
snap    2.62
snapd   2.62
series  16
ubuntu  22.04
kernel  6.5.0-27-generic
```

**Comment by dkolkin** [comment]
I'm having the exact same problem running BW on Ubuntu 20.04.6LTS. None of the recommended options offered by the BW team seemed to help. I'm not an expert, and just wondering if the issue might stem from the fact that the application was installed from snap.

**Comment by jlf0dev** [comment]
Closing for now to route these linux token storage issues through #8651. Thanks for the report!
